sklearn/metrics/classification.py

```diff
@@ -596,72 +597,135 @@ def jaccard_similarity_score(y_true, y_pred, normalize=True,
     y_pred : 1d array-like, or label indicator array / sparse matrix
         Predicted labels, as returned by a classifier.
 
+    labels : list, optional
+        The set of labels to include when ``average != 'binary'``, and their
+        order if ``average is None``. Labels present in the data can be
+        excluded, for example to calculate a multiclass average ignoring a
+        majority negative class, while labels not present in the data will
+        result in 0 components in a macro average. For multilabel targets,
+        labels are column indices. By default, all labels in ``y_true`` and
```

Comment:
I think the description for the `label -> order of things` is not explained clearly enough here.

Reply:
This is only true of average=None. The text is the same as for precision_score, etc, so I would like to avoid changing it here.